obsreport/obsreport_pipeline.go

```diff
@@ -0,0 +1,69 @@
+// Copyright The OpenTelemetry Authors
```

Comment:
There is a lot happening within one file, is it possible to seperate it out into one individual files?

Reply:
When I rebased, the metric definitions were moved to `internal/obsreportconfig/obsmetrics`, so this file was split into definitions and functions